ourhexenv.py

```diff
@@ -93,8 +93,8 @@ def __init__(self, board_size=5, sparse_flag=True):
             }
         else:
             self.reward_mapping: Dict[str, int] = {
-                "WIN": (self.board_size*self.board_size)//2,
```

Comment:
Let's do max_reward = self.board_size*self.board_size)//2

WIN: math.floor(max_reward), Loose: -math.ceil(max_reward)

Reply:
i like that, coming right up